nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/PropertyManager.java

```diff
@@ -753,13 +753,7 @@ void setupItems() {
                             ArrayList defaultList = (ArrayList) parMap.get("default");
                             //ListOperationItemTypeSelector typeSelector = new ListOperationItemTypeSelector(stringListener, (String) listTypes.get(0), listTypes, op, "listType", parDesc);
                             ListOperationItemTypeSelector typeSelector = null;
-
-                            if (listTypes == null || listTypes.isEmpty()) { // just a default list, so treat them as real values.
-                                lstItem = new ListOperationItem(listListener, defaultList, listTypes, op, name, parDesc, typeSelector);
-                            } else {
-                                lstItem = new ListOperationItem(listListener, defaultList, listTypes, op, name, parDesc, typeSelector);
-                            }
-
+                            lstItem = new ListOperationItem(listListener, defaultList, listTypes, op, name, parDesc, typeSelector);
```

Comment:
The ListOperationItem was created the same way regardless of the value of list items, so I collapsed the if statement. In ListOperationItem constructor, listTypes is assigned to ListOperationItem.listTypes which doesn’t seem like it is used anywhere (
https://github.com/nanalysis/nmrfx/blob/7d9b8e15b120fd9ddb133501a913dcd9801e67fe/nmrfx-utils/src/main/java/org/nmrfx/utils/properties/ListOperationItem.java#L67, 
https://github.com/nanalysis/nmrfx/blob/7d9b8e15b120fd9ddb133501a913dcd9801e67fe/nmrfx-utils/src/main/java/org/nmrfx/utils/properties/ListOperationItem.java#L45
)
should this variable be removed from ListOperationItem?

Reply:
I would remove it - the less code we keep, the less bugs we risk. If it was useful, someone would have noticed that it didn't do anything since at least 2018.